doc/ASSET_API.rst

```diff
@@ -0,0 +1,113 @@
+============================
+Vega Strike Engine Asset API
+============================
+
+This document provides a changelog of the Vega Strike Engine Asset API.
+
+What is the Vega Strike Engine Asset API?
+-----------------------------------------
+
+The Vega Strike Engine Asset API is the Python-based bindings used by the Game Assets.
+
+The Vega Strike Engine itself provides the ability for Game Assets, such as
+Vega Strike: Upon the Coldest Sea, to create a 3D Space-based game. The Game Assets
+and the Vega Strike Engine interface using the Vega Strike Engine Asset API.
+
+Vega Strike Engine Versioning
+-----------------------------
+
+As of version 0.8.0 the Vega Strike Engine Asset API, aka Asset API, provides an interface
+for determining the versions information related to the Vega Strike Engine including:
+
+- the official version fo the Vega Strike Engine (f.e `0.8.0`)
+- the version of the Asset API implemented by the Vega Strike Engine using a simple incrementing
+  integer value.
+
+For simplicity, the Asset API Version is a simple incrementing interger value with a behavior
+like that of the Android SDK version. The Asset API and the Vega Strike Engine versions have a
+loose relationship. Game Assets should use the Asset API Version as the basis for the
+determining available functionality.
```

Comment:
for the determining available functionality:
Did you mean 'for determining the available functionality' ?